frontend/src/components/App/index.js

```diff
@@ -16,14 +17,16 @@ class App extends React.Component {
   render() {
     return (
       <AuthProvider>
-        <Router>
-          <div>
-            <Route exact path={ROUTES.LANDING} component={LandingPage} />
-            <Route path={ROUTES.SIGN_IN} component={SignInPage} />
-            <PrivateRoute path={ROUTES.VIEW_TRIPS} component={ViewTripsPage} />
-            <PrivateRoute path={ROUTES.VIEW_ACTIVITIES} component={ViewActivitiesPage} />
-          </div>
-        </Router>
+        <AuthUtilsConsumer>
+          <Router>
+            <div>
+              <Route exact path={ROUTES.LANDING} component={LandingPage} />
+              <Route path={ROUTES.SIGN_IN} component={SignInPage} />
+              <PrivateRoute path={ROUTES.VIEW_TRIPS} component={ViewTripsPage} />
+              <PrivateRoute path={ROUTES.VIEW_ACTIVITIES} component={ViewActivitiesPage} />
```

Comment:
I don't know why this isn't on the previous version of this document either, but you do need the `+ "/:tripId"` or else we can't pass arguments from view_trips to view_activities.

Reply:
+1  I asked about this this on PR #45 I believe too

Reply:
Added the URL parameter. 👍